grammar-tools/MySQLParser.g4

```diff
@@ -2965,9 +2965,9 @@ simpleExpr: %simpleExpr_collate (CONCAT_PIPES_SYMBOL %simpleExpr_collate)*;
     literal                                                                                              # simpleExprLiteral
     | sumExpr                                                                                            # simpleExprSum
     | variable (equal expr)?                                                                             # simpleExprVariable
-    | functionCall                                                                                       # simpleExprFunction
-    | runtimeFunctionCall                                                                                # simpleExprRuntimeFunction
-    | columnRef jsonOperator?                                                                            # simpleExprColumnRef
+    /*| functionCall                                                                                       # simpleExprFunction*/
+    /*| runtimeFunctionCall                                                                                # simpleExprRuntimeFunction*/
+    /*| columnRef jsonOperator?                                                                            # simpleExprColumnRef*/
```

Comment:
can we remove these commented out lines?

Reply:
@adamziel These are all related to the `FIX` comments discussed in https://github.com/Automattic/sqlite-database-integration/pull/1#discussion_r1939068451. I think we can remove those when we're ready to remove the fix comments.